docs/sources/mimir/configure/configure-object-storage-backend.md

```diff
@@ -108,7 +108,7 @@ common:
   storage:
     backend: azure
     azure:
-      account_key: "${SWIFT_ACCOUNT_KEY}" # This is a secret injected via an environment variable
+      account_key: "${AZURE_ACCOUNT_KEY}" # This is a secret injected via an environment variable
```

Comment:
```suggestion
      account_key: "${AZURE-ACCOUNT-KEY}" # This is a secret injected via an environment variable
```

Reply:
`${AZURE-ACCOUNT-KEY}` isn't going to work, since it's supposed to be a shell variable (and `-` is an operator in the shell language).